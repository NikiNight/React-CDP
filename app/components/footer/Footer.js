import React, { Component } from 'react';
import Logo from '../common/Logo';

const Footer = () => {
    return (
        <footer className="footer">
            <div className="wrapper">
                <Logo/>
            </div>
        </footer>
    )
}

export default Footer;