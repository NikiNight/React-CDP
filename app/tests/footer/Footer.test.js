import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './../../components/footer/Footer';

test('matches snapshot', () => {
  const tree = renderer.create(<Footer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});