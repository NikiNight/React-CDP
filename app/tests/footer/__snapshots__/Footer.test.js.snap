// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="wrapper"
  >
    <a
      className="logo"
      href="#"
    >
      netflixroulette
    </a>
  </div>
</footer>
`;
